api/v2/endpoints/post-identity-map.md

```diff
@@ -123,6 +123,54 @@ A successful decrypted response returns the UID2s and salt bucket IDs for the sp
 }
 ```
 
+When some identifiers are considered invalid we will use "unmapped" list to deliver them. In this case, the response status remains "success".
```

Comment:
If some identifiers are considered invalid, they are included in the response in an "unmapped" list. In this case, the response status is still "success". If all identifiers are mapped, the "unmapped" list is not included in the response.

Reply:
great, changed!